go.mod

```diff
@@ -68,8 +69,9 @@ require (
 	github.com/NYTimes/gziphandler v1.1.1 // indirect
 	github.com/bronze1man/goStrongswanVici v0.0.0-20190828090544-27d02f80ba40 // indirect
 	github.com/containerd/cgroups v0.0.0-20200710171044-318312a37340
-	github.com/containerd/containerd v1.4.1
+	github.com/containerd/containerd v1.5.0-beta.0.0.20210122062454-5a66c2ae5cec
```

Comment:
github.com/k3s-io/containerd  needs to be synced for this?

Reply:
Stargz Snapshotter v0.4.1 is tested with containerd v1.4.3 so `github.com/k3s-io/containerd v1.4.3-k3s3` seems fine.

https://github.com/containerd/stargz-snapshotter/blob/v0.4.1/Dockerfile#L15